### ISSUE TITLE
Suppress default params table for a function

### ISSUE THREAD

**Issue by telnet2** [vmarchaud/jsdoc-http-plugin#11]
I have a following code piece documented.

```javascript
 /**
     * Requires `customerOrderId` and `role` parameters.
     * @path {PUT} /eta/update/:customerOrder/:role
     * @params :customerOrderId
     * @params :role
     */
    static async updateTimer(req, res)
```

I really like the output of parameters related to HTTP routing. However, I would like to suppress the parameters section, which is in fact not relevant to the function signature. Is there such an option?

![image](https://user-images.githubusercontent.com/313920/47189812-37cf0480-d2f3-11e8-9bc9-e60e3ef86217.png)

This plugin is very useful!

**Comment by vmarchaud** [comment]
I don't believe it's possible because `@params` are used by the standard JSDoc also and we cannot override this behavior. You could however make a script that remove them from your docs with some regex i believe
